release notes/v0.46.0.md

```diff
@@ -89,6 +141,24 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [#3136](https://github.com/grafana/k6/pull/3136) Updates k6 Go dependencies.
 - [#3131](https://github.com/grafana/k6/pull/3131) Updates our Pull Request template to be more structured and include a checklist for contributors.
 - [#3177](https://github.com/grafana/k6/pull/3177) Updates the version of golangci-lint we use to the latest version.
+- [browser#849](https://github.com/grafana/xk6-browser/pull/849) Updates `nil` session log warn to debug level in page attach.
+- [browser#881](https://github.com/grafana/xk6-browser/pull/881) Adds a new `browser_http_req_failed` metric.
+- [browser#889](https://github.com/grafana/xk6-browser/pull/889) Refactors `isRemoteBrowser` check.
+- [browser#889](https://github.com/grafana/xk6-browser/pull/899) Refactors pid registry.
+- [browser#902](https://github.com/grafana/xk6-browser/pull/902) Refactors registry back into browser package.
+- [browser#904](https://github.com/grafana/xk6-browser/pull/904) Fixes a race condition in Web Vitals unit test.
+- [browser#906](https://github.com/grafana/xk6-browser/pull/906) Parses the scenarios JSON definition from the `K6_INSTANCE_SCENARIOS` environment variable.
+- [browser#918](https://github.com/grafana/xk6-browser/pull/918) Replaces the usage of `os.LookupEnv` for k6 `LookupEnv` and abstracts environment variables lookup.
+- [browser#923](https://github.com/grafana/xk6-browser/pull/923) Removes the browser module exported version field.
```

Comment:
Should this go in breaking changes?

Reply:
Yep you are right, it was an exported element after all. Let me fix it.

Reply:
Fixed in https://github.com/grafana/k6/pull/3260/commits/86e3f0bfeb8b84eee51a5c3640de4e6ce73d0449.